cmd/bosun/sched/check_test.go

```diff
@@ -258,6 +258,68 @@ Loop:
 	}
 }
 
+// TestCheckNotifyUnknownDefault tests the default unknownTemplate.
+func TestCheckNotifyUnknownDefault(t *testing.T) {
+	s := new(Schedule)
+	nc := make(chan string, 1)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ah, no, that won't work in this case because unknown notifications are computed separately and don't exist in a usable form by tests.